src/main/java/racing/Car.java

```diff
@@ -0,0 +1,52 @@
+package racing;
+
+import java.util.Random;
+
+public class Car {
+    private int distance;
+    private String carName;
+
+    public int getDistance() {
+        return distance;
+    }
+
+    public void setDistance(int distance) {
+        this.distance = distance;
+    }
+
+    public String getCarName() {
+        return carName;
+    }
+
+    public void setCarName(String carName) {
+        if(carName.length() > 5) {
+            System.out.println("자동차 이름은 5글자를 초과할 수 없습니다.");
+            throw new IllegalArgumentException();
+        }
```

Comment:
carName 제한에 대한 검증 👍
`System.out.println` 를 통해서 예외 메세지를 전달하고 있는데요. 현재는 콘솔 환경이라 문제가 없어 보일 수 있지만 웹 환경으로 바뀐다면 이 부분이 유효할까요? 예외를 던지면서 메세지도 함께 담아 주는 것은 어떨까요? 

```suggestion
        if(carName.length() > 5) {
            throw new IllegalArgumentException("자동차 이름은 5글자를 초과할 수 없습니다.");
        }
```